cloudformation/OpenMapKitServer.template.js

```diff
@@ -0,0 +1,178 @@
+const cf = require('@mapbox/cloudfriend');
+
+const Parameters = {
+  GitSha: {
+    Type: 'String',
+    Description: 'GitSha for this stack'
+  },
+  ELBSecurityGroup: {
+    Description: 'Security Group for the ELB',
+    Type: 'String'
+  },
+  ELBSubnets: {
+    Description: 'ELB subnets',
+    Type: 'String'
+  },
+  EC2SecurityGroup: {
+    Description: 'EC2 security group',
+    Type: 'String'
+  },
+  S3Bucket: {
+    Description: 'S3 bucket',
+    Type: 'String'
+  }
+};
+
+const Resources = {
+  OpenMapKitServerASG: {
+    DependsOn: 'OpenMapKitServerLaunchConfiguration',
+    Type: 'AWS::AutoScaling::AutoScalingGroup',
+    Properties: {
+      AutoScalingGroupName: cf.stackName,
+      Cooldown: 300,
+      MinSize: 0,
+      DesiredCapacity: 1,
+      MaxSize: 1,
+      HealthCheckGracePeriod: 300,
+      LaunchConfigurationName: cf.stackName,
+      LoadBalancerNames: [ cf.ref('OpenMapKitServerLoadBalancer') ],
+      HealthCheckType: 'EC2',
+      AvailabilityZones: cf.getAzs(cf.region)
+    }
+  },
+  OpenMapKitServerScaleUp: {
+      Type: "AWS::AutoScaling::ScalingPolicy",
+      Properties: {
+        AutoScalingGroupName: cf.ref('OpenMapKitServerASG'),
+        PolicyType: 'TargetTrackingScaling',
+        TargetTrackingConfiguration: {
+          TargetValue: 85,
+          PredefinedMetricSpecification: {
+            PredefinedMetricType: 'ASGAverageCPUUtilization'
+          }
+        },
+        Cooldown: 300
+      }
+  },
+  OpenMapKitServerLaunchConfiguration: {
+    Type: 'AWS::AutoScaling::LaunchConfiguration',
+      Properties: {
+        IamInstanceProfile: cf.ref('OpenMapKitServerEC2InstanceProfile'),
+        ImageId: 'ami-0e4372c1860d7426c',
+        InstanceType: 'm3.large',
```

Comment:
Currently we are using `t2.medium` to the Nepal instance.

Reply:
Noted. 🙂